compiler/rustc_lint/src/nonstandard_style.rs

```diff
@@ -177,6 +182,7 @@ impl EarlyLintPass for NonCamelCaseTypes {
             // trait impls where we should have warned for the trait definition already.
             ast::ItemKind::Impl(box ast::Impl { of_trait: None, items, .. }) => {
                 for it in items {
+                    // FIXME: this doesn't respect `#[allow(..)]` on the item itself.
```

Comment:
why not fix this too? you have the right machinery to do it, right?

Reply:
Whoops, posted at the same time.